tests/unittests/conftest.py

```diff
@@ -50,6 +50,7 @@ def m_gpg():
         ("symlink", 2),
         ("stat", 1),
         ("scandir", 1),
+        ("utime", 2),
```

Comment:
`os.utime()` needs to be trapped and wrapped because otherwise tests (such as `TestIsIBMProvisioning::test_config_with_new_log`) using the `fake_filesystem` fixture and the helper test function `populate_dir_with_ts` will fail because this function uses `os.utime()`